### ISSUE TITLE
Quality icons?

### ISSUE THREAD

**Issue by WanderingFox** [doadin/Baggins#83]
Would it be possible to add quality icons to items (in the same way item upgrade arrows are added)?

Like how the profession window shows them:
![image](https://user-images.githubusercontent.com/2967926/204629018-6b92b156-8c9f-464c-bc1d-3bf701558b7b.png)

**Comment by doadin** [comment]
I think so but im not sure how to get this info from the game, Ill look into it.

**Comment by WanderingFox** [comment]
The icons appear to be in the atlass `professionqualityicons` as:

- `Professions-Icon-Quality-Tier1-Inv`
- `Professions-Icon-Quality-Tier2-Inv`
- `Professions-Icon-Quality-Tier3-Inv`
- `Professions-Icon-Quality-Tier4-Inv`
- `Professions-Icon-Quality-Tier5-Inv`

Not sure how to get the quality of the item in the bag though.

![image](https://user-images.githubusercontent.com/2967926/204956867-662bad8a-a682-4c46-991b-e2b99127b756.png)

**Comment by WanderingFox** [comment]
Looks like it's done something like this in the blizz source (not sure how much is actually needed for baggins though since this is ripped out of the profession ui):

```
quality = C_TradeSkillUI.GetItemReagentQualityByItemInfo(itemIDOrLink) or  C_TradeSkillUI.GetItemCraftedQualityByItemInfo(itemIDOrLink);
button.ProfessionQualityOverlay = button:CreateTexture(nil, "OVERLAY");
button.ProfessionQualityOverlay:SetPoint("TOPLEFT", -3, 2);
button.ProfessionQualityOverlay:SetDrawLayer("OVERLAY", 7);
local atlas = ("Professions-Icon-Quality-Tier%d-Inv"):format(quality);
button.ProfessionQualityOverlay:SetAtlas(atlas, TextureKitConstants.UseAtlasSize);
ItemButtonMixin.UpdateCraftedProfessionsQualityShown(button);
```

**Comment by doadin** [comment]
Added: e6a67bd8b050398c52b01155ad4fef9dccc9945d
Included in v4.6.9 . Let me know if you have any issues with it. Thanks!
